product_docs/docs/epas/14/epas_platform_support/index.mdx

```diff
@@ -18,6 +20,28 @@ EDB Postgres Advanced Server v14 Beta 1 supports installations on the following
 
 See [Product Compatibility](https://www.enterprisedb.com/platform-compatibility) for additional information.
 
+## Hardware Requirements
+
+The following installation requirements assume you have selected the default options during the installation process. The minimum hardware required to install and run EDB Postgres Advanced Server are:
+
+-   1 GHz processor
+-   2 GB of RAM
+-   512 MB of HDD
+
+Additional disk space is required for data or supporting components.
+
+## Software Prerequisites
+
+**User Privileges**
+
+To perform an Advanced Server installation on a Windows system you must have administrator privilege. If you are installing Advanced Server on a Windows system that is configured with `User Account Control` enabled, you can assume sufficient privileges to invoke the graphical installer by right clicking on the name of the installer and selecting `Run as administrator` from the context menu.
+
+To perform an Advanced Server installation on a Linux system you must have superuser or administrator or sudo privileges.
```

Comment:
Since Linux is our primary platform, move the Linux paragraph above the Windows one.

Reply:
Also, be consistent with the EPAS product name, we are switching between the full name and the short name in this topic.